book/command_signature.md

```diff
@@ -1,24 +1,22 @@
 # Command signature
 
-nu commands contain a signature section, take [`str distance`](/commands/docs/str_distance.md) as example, the signature is like this:
+nu commands can be given explicit signatures; take [`str distance`](/commands/docs/str_distance.md) as an example, the signature is like this:
 
 ```nu
-Signatures(Cell paths are supported):
-  <string> | str distance <string> -> <int>
+def "str distance" [compare-string: string, ...rest: cell-path]: string -> int { }
```

Comment:
to follow completely the i/o types of `str distance`, i think it should be
```suggestion
def "str distance" [compare-string: string, ...rest: cell-path]: [string -> int, table -> table, record -> record] { }
```

Reply:
EDIT: aah i see you mention the full signature below :+1: 
maybe just add here that this is a simplified signature?